admin/packaging.py

```diff
@@ -921,6 +932,9 @@ def omnibus_package_builder(
                     # SystemD configuration
                     package_files.child('systemd'):
                         FilePath('/usr/lib/systemd/system'),
+                    # Upstart configuration
```

Comment:
Yes, I think it is unusual, but it is the way it works right now. I have created issue FLOC-1736 to look at fixing it.